configs/swin/upernet_swin_large_patch4_window7_512x512_pretrain_224x224_22K_160k_ade20k.py

```diff
@@ -0,0 +1,15 @@
+_base_ = [
+    'upernet_swin_tiny_patch4_window7_512x512_160k_ade20k_'
+    'pretrain_224x224_1K.py'
+]
+checkpoint_file = 'https://download.openmmlab.com/mmsegmentation/v0.5/pretrain/swin/swin_large_patch4_window7_224_22k_20220412-aeecf2aa.pth'  # noqa
+model = dict(
+    backbone=dict(
+        init_cfg=dict(type='Pretrained', checkpoint=checkpoint_file),
+        pretrain_img_size=224,
+        embed_dims=192,
+        depths=[2, 2, 18, 2],
+        num_heads=[6, 12, 24, 48],
+        window_size=7),
```

Comment:
May use window_size=12 (i.e., pretrain model use window_size =12) and configs/_base_/datasets/ade20k_640x640.py (22k pretrain model always use this config) to retrain again.

Reply:
> May use window_size=12 (i.e., pretrain model use window_size =12) and configs/_base_/datasets/ade20k_640x640.py (22k pretrain model always use this config) to retrain again.

Thx, I would retrain it later.

Reply:
My fault，it had been done.